src/main/java/uk/gov/hmcts/reform/cpo/payload/UpdateCasePaymentOrderRequest.java

```diff
@@ -1,50 +1,44 @@
 package uk.gov.hmcts.reform.cpo.payload;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
-import lombok.AllArgsConstructor;
 import lombok.Getter;
-import uk.gov.hmcts.reform.cpo.validators.annotation.ValidCaseId;
-import uk.gov.hmcts.reform.cpo.validators.annotation.ValidCpoId;
 import uk.gov.hmcts.reform.cpo.validators.ValidationError;
+import uk.gov.hmcts.reform.cpo.validators.annotation.ValidCpoId;
 
 import javax.validation.constraints.NotEmpty;
-import javax.validation.constraints.NotNull;
 import java.time.LocalDateTime;
+import java.util.UUID;
 
 @Getter
-@AllArgsConstructor
 @ApiModel("Update Case Payment Order Request")
-public class UpdateCasePaymentOrderRequest {
+public class UpdateCasePaymentOrderRequest extends CreateCasePaymentOrderRequest {
```

Comment:
minor: 
we are using inheritance here purely for code reuse I guess. Since conceptually an update request is not a create request. I would avoid doing this.
Also, it could limit our flexibility in the future. Imagine for example a new property gets introduced on the create request but for which we don't want to make it updatable...

Reply:
Resolved in 6760f7ddf235c1c4bdef814215dc87ff7a6aa96f